src/test/run-make/incr-prev-body-beyond-eof/Makefile

```diff
@@ -2,6 +2,7 @@ include ../../run-make-fulldeps/tools.mk
 
 # FIXME https://github.com/rust-lang/rust/issues/78911
 # ignore-32bit wrong/no cross compiler and sometimes we pass wrong gcc args (-m64)
+# ignore-riscv64 unrecognized command line option '-m64'
```

Comment:
Would be good to know why this argument gets to the linker in the first place – the riscv64 target does not appear to specify such an argument the way x86_64 targets do, for instance.

Reply:
Yeah, I'd prefer to not patch over this - we should figure out why this is getting through. It looks like it's plausibly also a problem on 32-bit targets for some reason.

Reply:
It turns out this was the same issue as for 32-bit platforms. The problem was that we didn't call rustc with `--target` but did specify the linker for the target platform.